lib/memory-accounting/src/api.rs

```diff
@@ -69,8 +69,9 @@ impl MemoryRegistryState {
             component_usage.insert(
                 component_name.to_string(),
                 ComponentUsage {
-                    minimum_required_bytes: bounds.self_minimum_required_bytes,
-                    firm_limit_bytes: bounds.self_firm_limit_bytes,
+                    // TODO: figure out what's actually going on here, this might not be a valid change
```

Comment:
Regarding the TODO... what behavior did you see that led you to make this change?

Reply:
No behavior that made me suspicious, just that I was moving quickly through these changes and didn't verify that there wasn't some semantic difference in this conversion that I'd neglected.

Reply:
Ahh, gotcha. 👍🏻